lnd-channels-backup-dependencies.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# run as root
+sudo su -
+
+# update system
+echo "Updating system"
+apt update && apt upgrade -y && apt autoremove
+
+# generate btcpayserver ssh key
+echo "Generating SSH key"
+ssh-keygen -o -a 100 -t ed25519 -f /root/.ssh/id_ed25519_btcpayserver -N ''
+
+# install inotify and rsync
+echo "Installing inotify and rsync"
+apt install -y inotify-tools rsync
+
+# install rclone
+echo "Installing rclone"
+curl https://rclone.org/install.sh | bash
```

Comment:
don't install if already installed.

Reply:
why not rsync?

Reply:
rsync does the remote server part, rclone does the cloud 'cause it is suited for dealing with all these clouds filesystems, permissions, etc so much better

Reply:
rclone has been well maintained, supports 30+ clouds and growing, is regular user-friendly with auto config, has built-in crypt option for advanced users